test-common/test-common-api/src/main/java/org/apache/kafka/common/test/api/Quarantined.java

```diff
@@ -0,0 +1,40 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.kafka.common.test.api;
+
+import org.junit.jupiter.api.Tag;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+@Target({ElementType.TYPE, ElementType.METHOD})
+@Retention(RetentionPolicy.RUNTIME)
+@Tag("quarantined")
+public @interface Quarantined {
```

Comment:
Could you please update KIP based on this new tag? the tag name in KIP is `Flaky` rather than `Quarantined`

Reply:
Actually, I wonder if `@Flaky` is better for the annotation. 

The "quarantine" as a concept includes flaky tests as well as new integration tests, so it might make sense to keep the flaky annotation. 

```
@Tag("quarantined")
public @interface Flaky 
```

or 
```
@Tag("flaky")
public @interface Flaky 
```

WDYT?

Reply:
> @Tag("flaky")
public @interface Flaky 

I prefer this one since it’s shorter and has been part of Kafka for a long time.